### ISSUE TITLE
Create a NuGet package for the library

### ISSUE THREAD

**Issue by JeremyAnsel** [cyotek/Cyotek.Drawing.BitmapFont#1]

**Comment by cyotek** [comment]
Thanks for the suggestion - it's incoming, I had wanted to review the code first (it's been untouched for a couple of years), but it should be added in the next couple of days.

Regards;
Richard Moss

**Comment by cyotek** [comment]
Hello,

I've published [two packages](https://www.nuget.org/packages/Cyotek.Drawing.BitmapFont).

The first is rebuilt from the original source, untouched and unsigned.

The second is signed and has some improvements, keeping the same public API. Annoying as if I were writing it now I would make different decisions and I think some of the API it is poor. However, if it's being used in the wild then it's a bit late! Assuming I ever get time to get back to my own game development activities, I may branch this library to a v2 and change it around some.

Oh, and the second version hasn't been tested much, as of course there's no tests and no documentation. Hopefully I haven't introduced anything, but I did tinker some of the string processing to make it more efficient so it's possible.

I also experimented with creating GitHub releases as I haven't looked at these before.

Regards;
Richard Moss
